languages/es.json

```diff
@@ -0,0 +1,253 @@
+				"rollAttributeTest": "Tirada de prueba de caracter�stica",
+				"rollUsageDie": "Tirada de Dado de uso",
+				"spellAlreadyCast": "El conjuro ya se ha lanzado al menos una vez",
+				"unprepared": "No preparado",
+				"unpreparePrayer": "No preparar esta plegaria",
+				"unprepareSpell": "No preparar este conjuro"
+			}
+		},
+		"initiative": {
+			"roll": "Tirando DES para la iniciativa."
+		},
+		"magic": {
+			"kinds": {
+				"prayer": "Plegaria",
+				"spell": "Conjuro"
+			}
+		},
+		"messages": {
+			"attributeTestFailed": "La prueba fue un fallo.",
+			"attributeTestSuccess": "La prueba fue un �xito.",
+			"attacking": "%NAME% ataca con %WEAPON%",
+			"attackMiss": "...pero falla.",
+			"blurbs": {
+				"attacksWith": "Ataca con",
+				"casts": "Lanza el conjuro",
+				"castsRitual": "Intenta lanzarlo como un ritual",
+				"damageInflicted": "Da�o infligido",
+				"spellLost": "Este conjuro ya no est� preparado ni disponible para su uso.",
+				"spellMiscast": "El conjuro no se lanza y el lanzador debe tirar en la tabla de Efectos secundarios m�gicos.",
+				"usageDie": "Tirando el Dado de uso para",
+				"usageDieDowngrade": "El Dado de uso desciende a:",
+				"usageDieExhausted": "El Dado de uso se ha agotado para:"
+			},
+			"criticalHit": "...y causa un impacto cr�tico. Tirando da�o...",
+			"errors": {
+				"supplyDepleted": "No se puede restablecer el dado de uso para %ITEM% dado que el recurso se ha agotado.",
+				"usageDieExhausted": "El Dado de uso de %ITEM% ya est� agotado."
+			},
+			"labels": {
+				"buttons": {
+					"rollDamage": "Tirada de da�o"
+				},
+				"critical": "�Impacto cr�tico!",
+				"failure": "Fracaso",
+				"hit": "�Impacto!",
+				"miss": "Fallo",
+				"success": "�xito"
+			},
+			"normalHit": "...e impacta. Tirando da�o...",
+			"reducingUsageDie": "Dado de uso descendido a %DIE%.",
+			"rollingAttributeTest": "%NAME% est� realizando una prueba de caracter�stica de %ATTRIBUTE%...",
+			"rollingUsageDie": "Tirando el Dado de uso para %NAME%.",
+			"titles": {
+				"attackRoll": "Tirada de ataque",
+				"attributeTest": "Prueba: %ATTRIBUTE%",
+				"damageRoll": "Tirada de da�o",
+				"failure": "Fallo",
+				"formula": "Tirada",
+				"roll": "Resultado",
+				"success": "�xito",
+				"test": "Prueba:",
+				"usageDie": "Dado de uso"
+			},
+			"usageDieExhausted": "El Dado de uso para tu %NAME% est� agotado."
+		},
+		"ranges": {
+			"close": "Inmediato",
+			"nearby": "Cercano",
+			"faraway": "Lejano",
+			"distant": "Distante",
+			"special": "Especial"
+		},
+		"rarities": {
+			"common": "Com�n",
+			"exotic": "Ex�tico",
+			"rare": "Raro"
+		},
+		"rolls": {
+			"advantage": "Ventaja",
+			"disadvantage": "Desventaja",
+			"modifier": "Modificador de la tirada:"
+		},
+		"settings": {
+			"options": {
+				"randomizeHP": {
+					"blurb": "Indicates whether creatures added to scenes will have their HP randomly generated.",
+					"title": "Randomize Creature HP"
+				}
+			}
+		},
```

Comment:
A few more translations needed here @WallaceMcGregor.

Reply:
"blurb": "Indica si las criaturas añadidas a las escenas tendrán sus PG generados aleatoriamente."
"title": "Aleatorizar PG de la criatura"

Reply:
Ta. Got work out a few layout issues but I think I should have this out in a few days.